build.gradle

```
@@ -3,8 +3,8 @@ plugins {
     id 'maven-publish'
 }
 
-sourceCompatibility = JavaVersion.VERSION_16
-targetCompatibility = JavaVersion.VERSION_16
+sourceCompatibility = "17"
+targetCompatibility = "17"
```

```suggestion
targetCompatibility = JavaVersion.VERSION_17
```
Just a minor nitpick